redirects/15.0.txt

```diff
@@ -1,5 +1,8 @@
 # applications/finance
 
+applications/finance/accounting/others/multicurrencies/exchange.rst # deleted as it was exactly same content as multicurrencies/multicurrency.rst
```

Comment:
Making a rule is simple (and no need to add explanations as to why).
On a single line:
old path `app/.../.rst`
a space
new path `app/.../.rst`